docs/testing.md

```diff
@@ -0,0 +1,10 @@
+# Testing instructions
+
+### Using Maestro E2E tests
+1. Install Maestro by following [these instuctions](https://maestro.mobile.dev/). This will help us run the E2E tests.
+
+
+### Using Flashlight for Performance Testing
+1. Make sure Maestro is installed (optional: only for auomated testing)
+2. Install Flashlight by following [these instructions](https://flashlight.dev)
+3. 
```

Comment:
How do these tools relate to Detox?

Do we only use them for perf?

Can you add instructions on actually running these in our repo? E.g. which yarn scripts, how to interpret results

Reply:
Yes. I am finishing writing instructions right now.

Reply:
Done

Reply:
It can be used for other things than perf too. Added all instructions. Please open a new comment if its still confusing.